TideApp/TideApp/Domain/Model/WeatherResponse.swift

```diff
@@ -45,17 +45,18 @@ struct WeatherData: Decodable {
       enum CodingKeys: String, CodingKey {
         case tideData = "tide_data"
       }
-      
-      enum TideType: String, Codable {
-        case high = "HIGH"
-        case low = "LOW"
-      }
 
       // MARK: - TideData
       struct TideData: Identifiable, Decodable {
+
+        enum `Type`: String, Codable {
```

Comment:
minor, my 2 cents: I think TideType was nice because was more explanative about what type of data is, what I mean is Type, in a TideData, seems to represent the TideData Type, but instead is a TideType.

Reply:
I did think this as well, but I'm not too bothered either way

Reply:
Yeah, I get what you mean. I think its nice to have it nested in TideData as thats how the API represents it. Thoughts on `TideData.TypeType`?